python/lsst/obs/test/testCamera.py

```diff
@@ -63,7 +64,7 @@ def __init__(self):
         cameraTransformMap = cameraGeom.CameraTransformMap(cameraGeom.FOCAL_PLANE,
                                                            {cameraGeom.PUPIL: pupilToFocalPlane})
         detectorList = self._makeDetectorList(pupilToFocalPlane)
-        cameraGeom.Camera.__init__(self, "test", detectorList, cameraTransformMap)
+        cameraGeom.Camera.__init__(self, "test", detectorList, cameraTransformMap, PupilFactory)
```

Comment:
This is related to a comment in the `afw` changeset that if the `Camera` constructor can be defaulted to take the `PupilFactory` abstract base class, there would not need to be a change here.